javascript/exorelay/features/support/steps-given.ls

```diff
@@ -11,26 +11,15 @@ require! {
 module.exports = ->
 
 
-  @Given /^a hypothetical "([^"]*)" message$/, (message-name) ->
+  @Given /^a "([^"]*)" message$/, (message-name) ->
     global[message-name] = sinon.stub!
 
 
   @Given /^a new ExoRelay instance connecting to port (\d+)$/ (@exocom-port) ->
     @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = 'test-service'}
 
 
-  @Given /^an ExoRelay instance$/, (done) ->
-    @exo-relay = new ExoRelay {exocom-host: 'localhost', @exocom-port, role: @role = \test-service}
-      ..connect!
-      ..on 'online', ~>
-        wait-until (~> @exocom.received-messages.length), 10, ~>
-          if @exocom.received-messages |> any (.name is "exocom.register-service")
-            @exocom.reset!
-          done!
-      ..on 'error', (@error) ~>
-
-
-  @Given /^an ExoRelay instance called "([^"]*)"$/, (instance-name, done) ->
+  @Given /^an ExoRelay instance(?: called ".+")?$/, (done) ->
```

Comment:
In a follow up PR, can we please remove any use of `called ".*"` since that is unused